feat(pfd): AUTO BRAKE label (#6879)
flybywiresim/a32nx: .github/CHANGELOG.md

## 0.8.0

> 1. [PFD] Included Auto BRK messages in PFD as latest real life update - @brainshot (brainshot)

Reviewer: ```suggestion
1. [PFD] Added Auto BRK messages to the FMA - @brainshot (brainshot)
```
Slightly reworded.
